src/com/egeniq/utils/loader/TypefaceLoader.java

```diff
@@ -0,0 +1,43 @@
+package com.egeniq.utils.loader;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.util.Log;
+
+import java.util.Hashtable;
+
+/**
+ * Created by josvanegmond on 7/27/15.
+ *
+ * Class caching Typefaces to reduce loading time
+ * Original code from https://code.google.com/p/android/issues/detail?id=9904
+ */
+public class TypefaceLoader {
```

Comment:
Isn't this class the same as: https://github.com/egeniq/egeniq-android/blob/master/src/com/egeniq/utils/FontLoader.java ?